includes/cli/class-export.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Campaigns Export CLI command.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Campaigns\CLI;
+
+use \WP_CLI;
+use \WP_CLI_Command;
+use Newspack_Popups_Exporter;
+
+/**
+ * Campaigns Export CLI command.
+ */
+class Export extends WP_CLI_Command {
+
+	/**
+	 * Export Campaigns, Segments and Prompts.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--file=<file>]
+	 * : The name of the output file.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp newspack-campaigns export --file=newspack-campaigns.json
```

Comment:
> The command is newspack-popups and not newspack-campaigns

I was following the code comment, which should be updated: 

```suggestion
	 *     wp newspack-popups export --file=newspack-campaigns.json
```